Makefile

```diff
@@ -44,5 +48,5 @@ clean:
 ui-dev:
 	npm --prefix webserver/ install
 
-ui:
-	npm --prefix webserver/ run build
+ui: $(ui-dev)
```

Comment:
This appears to update all the npm packages each time you build the UI. I'd prefer not to do that because you never know what breaking change might come in from a vendor. This seems more like a manual thing to do as part of every minor release.